ibllib/qc/alignment_qc.py

```diff
@@ -10,6 +10,13 @@
 _log = logging.getLogger('ibllib')
 CRITERIA = {"PASS": 0.8}
 
+CRITERIA_BASE = {'CRITICAL': 4,

```

Comment:
This map is in [qc/base.py](https://github.com/int-brain-lab/ibllib/blob/56969ec8a78d8ccaa96edd96c1b945ef3034db67/ibllib/qc/base.py#L11-L17).  Could you import it from there so it's only one place?